### PR TITLE
chore(CheckboxGroup): extract and fix mocked JSON

### PR DESCRIPTION
**Pull request by vtsvetkov-splunk**
Mocked global config was not compliant with JSON schema. This PR fixing. Also, the whole config was extracted to separate JSON file for ease of validation and copy-pasting.
Fixes https://github.com/splunk/addonfactory-ucc-base-ui/pull/438#discussion_r1341994486

### CODE REVIEW

File: src/main/webapp/components/CheckboxGroup/checkboxGroupMocks.json
Comment (artemrys) [inline src/main/webapp/components/CheckboxGroup/checkboxGroupMocks.json:1]: We require at least some configuration page to be present in the globalConfig, you can check out https://github.com/splunk/addonfactory-ucc-generator/blob/main/tests/unit/testdata/valid_config_only_logging.json, I use it as a base.

Comment (vtsvetkov-splunk) [reply src/main/webapp/components/CheckboxGroup/checkboxGroupMocks.json:1]: Do you want me to make `configuration` property required for `pages` object?
https://github.com/splunk/addonfactory-ucc-base-ui/blob/67da73090ac3916e2872250708c845bed3d71825/src/main/webapp/schema/schema.json#L1290-L1303

Comment (artemrys) [reply src/main/webapp/components/CheckboxGroup/checkboxGroupMocks.json:1]: Interesting, let me double think it.

**Comment by vtsvetkov-splunk** [comment]
> Totally checked 163 files, valid: 3, invalid: 1, ignored: 159, fixed: 0 

>_>

File: src/main/webapp/components/CheckboxGroup/checkboxGroupMocks.json
Comment (vtsvetkov-splunk) [reply src/main/webapp/components/CheckboxGroup/checkboxGroupMocks.json:1]: Added in #442

**Pull request merged**